src/components/TvDetails/index.tsx

```diff
@@ -505,7 +538,7 @@ const TvDetails: React.FC<TvDetailsProps> = ({ tv }) => {
                     >
                       {intl.formatMessage(messages.watchtrailer)}
                     </ButtonWithDropdown.Item>
-                  ) : null}
+                  )}
```

Comment:
```suggestion
                  ) : null}
```

Reply:
this isn't a ternary statement, it causes an error :P

Reply:
Revert it back to the original logic with the ternary statement.  It was a ternary statement for a reason.  The only change you need to really make here is change instances of `plexUrl` or to `mediaUrl`.

Reply:
It was never a ternary statement, even in the current develop branch of this repo.

Reply:
converted to a ternary!